integration_tests/integration/referral-application.spec.ts

```diff
@@ -79,6 +79,35 @@ context('SignIn', () => {
     page.checkStatus('ap-type', 'Completed')
   })
 
+  it('Saves my answers', () => {
+    // Given I complete the questionaire
+    const page = ReferralApplicationTasklist.visit()
+    checkEligibility(page)
+    page.startSection('Select the type of AP Required')
+    const typeOfAp = Page.verifyOnPage(TypeOfAP)
+    typeOfAp.answerType('esap')
+    typeOfAp.saveAndContinue()
+    const esapReasons = Page.verifyOnPage(EsapReasons)
+    esapReasons.chooseReasons(['secreting', 'cctv'])
+    esapReasons.saveAndContinue()
+    cy.go(-3)
+    Page.verifyOnPage(ReferralApplicationTasklist)
+
+    // When I go back to previously answered sections
+    page.startSection('Check if the person is eligible')
+
+    // Then my previous answers aren't visible
+    cy.get(`input[name="referralReason"][value=likely]`).should('not.be.checked')
+
+    cy.go('back')
```

Comment:
I'm not quite sure I follow how these test case works out? How do these previously answered questions get cleared? And what does `cy.go(-3)` do?

I wonder if we could use `cy.session` with two named sessions (i.e. https://docs.cypress.io/api/commands/session#Conditionally-caching-a-session) to more easily demonstrate the fact that the answers disappear when the session is cleared.

Reply:
I thought the answers being cleared was the expected behaviour. For me locally on `main` if I complete a section and then go back to it the answers aren't saved but the state on the Referral Tasklist shows Completed. 

cy.go(-3) is the same as clicking the browser's 'back' button 3 times - I opted to go back instead of completing the form because I thought it was clearer what was going on but I can see now that it would probably be more helpful to complete the form. https://docs.cypress.io/api/commands/go 

I like the idea of having two named sessions to demonstrate what happens when a session is wiped, I will have a go at that